### ISSUE TITLE
Color use against own principles - Polaris Guide

### ISSUE THREAD

**Issue by brambilliet** [Shopify/polaris-react#1001]
<img width="262" alt="screenshot 2019-02-11 at 16 33 38" src="https://user-images.githubusercontent.com/17379429/52573662-e48d3e80-2e1a-11e9-9f93-045ebbdb332e.png">

Dark grey on Dark Indigo. Should be Light grey on Dark Indigo.

**Comment by chloerice** [comment]
Hey @brambilliet thanks for raising this. Can you give us more details on where exactly you're seeing this? Is this one of the components or are you seeing that somewhere in the partner dashboard or merchant admin?

**Comment by yourpalsonja** [comment]
@brambilliet Any follow-up on where we can find this issue?

**Comment by dpersing** [comment]
Hi @brambilliet: I'll close this issue for now since we aren't able to locate the example without some more context. Please let us know if you're able to provide more detail.
